### PR TITLE
Document Bio-Formats generated sources jobs

### PR DESCRIPTION
**Pull request by sbesson**
These jobs are used to commit the generated sources and expose model differences when modifying templates.

/cc @rleigh-dundee 

--no-rebase

### CODE REVIEW

File: contributing/ci-bio-formats.txt [hunk 1]
```diff
@@ -14,6 +14,10 @@ All jobs are listed under the :jenkinsview:`Bio-Formats` view tab of Jenkins.
 		* :term:`BIOFORMATS-5.1-latest`
 		*
 
+	-	* Generates the latest generated-sources
+		* :term:`BIOFORMATS-5.1-latest-generated-sources`
```
Comment (rleigh-dundee) [inline contributing/ci-bio-formats.txt]: Not sure what this job is for.  There no delta once it's merged!  What's the baseline for comparison here?

Comment (sbesson) [inline contributing/ci-bio-formats.txt]: @rleigh-dundee: correct, this job commits the baseline for comparison used in the merge job (and potentially for other jobs like BIOFORMATS-5.1-breaking-generated-sources). I will add a sentence to that effect to the job description if that makes sense.

**Comment by sbesson** [comment]
@rleigh-dundee: pushed more detailed description of the goal of the latest generated sources branch.

**Comment by rleigh-dundee** [comment]
This all looks fine to me.  Please merge.

**Pull request merged**